pallets/order-book/src/weights.rs

```diff
@@ -48,6 +74,14 @@ pub trait WeightInfo {
         Weight::zero()
     }
     fn cancel_limit_order() -> Weight {
+        // The functions for first/last expiring order are generated automatically.
+        // This implementation is not produced by benchmarking.
+        Self::cancel_limit_order_first_expiration().max(Self::cancel_limit_order_last_expiration())
```

Comment:
weights.rs should not be modified after generation to make easier automatization of benchmarking

Reply:
Now it's unaltered after generation.